buildenv/jenkins/jobs/infrastructure/signedOffByCheck

```diff
@@ -56,23 +56,28 @@ timeout(time: 6, unit: 'HOURS') {
                             def WRONG_COMMIT = true
                             def splitCommit = COMMIT.split("\\r?\\n")
                             def lastBlankLine = 0
-                            for (int i = splitCommit.size() - 1; i >= 0; i--) {
```

Comment:
We can probably be a little more strict on this regex. SHAs are always 40 characters and only contain numbers and letters a-f. It also seems to end in a period. Is there a set number of spaces at the beginning of the line?
Perhaps something like:
`^\s{4}This reverts commit [0-9a-f]{40}\.$`

Reply:
fixed

Reply:
fixed